scripts/watches/middlewares.py

```diff
@@ -0,0 +1,103 @@
+# -*- coding: utf-8 -*-
```

Comment:
do we need all those files ?

Reply:
Now - not really. If ever we decide to customize the scraper - then maybe. But can be without them now.